core/cap-0050.md

```diff
@@ -0,0 +1,183 @@
+```
+CAP: 0050
+Title: Smart Contract Signatures
+Working Group:
+    Owner: Jonathan Jove <@jonjove>
+    Authors:
+    Consulted:
+Status: Draft
+Created: 2022-05-10
+Discussion: TBD
```

Comment:
Since it seems these CAPs are being discussed on Discord, should this be:
```suggestion
Discussion: https://discord.com/channels/897514728459468821/935999748840751125/973675125201309726
```

Reply:
afaik Discord doesn't archive well and there will be many threads, making this a poor fit for historical purpose